index.js

```diff
@@ -0,0 +1,52 @@
+function getRandomIntInclusive(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+// console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+const diceTwo = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+function printDice(diceOneNumber, diceTwoNumber) {
+	let newClass = "img-0"+diceOneNumber;
+	let elm = document.getElementById("dice-01");
+
+	if(elm.className !== newClass){
+		elm.className = newClass;
+	}
+	let newClassTwo = "img-0"+diceTwoNumber;
+	let elmTwo = document.getElementById("dice-02");
+
+	if(elmTwo.className !== newClassTwo){
+		elmTwo.className = newClassTwo;
+	}
+}
+
+
+document.querySelector("button").addEventListener("click", event => {
+	event.preventDefault();
+	let diceOneNumber = dice.roll();
```

Comment:
You don't actually have to store the return of the dice into a variable. You can also just roll the dice and then pass `dice.value` into the `printDice()` function below, since the role method **actually** changes the `value` property on the dice object. Like this:

```javascript
dice.roll();
dice2.roll();

printDice(dice.value, dice2.value);
```